manifest.yaml

```diff
@@ -169,6 +165,14 @@ backup:
     io-format: yaml
 migrations:
   from:
+    "<0.14.2": # tor added in 0.14.2
```

Comment:
Now there are multiple versions of LND that have multiple migrations defined.. what happens in that case? Does the first matching migration run? Or all matching migrations?

Reply:
first